developing.adoc

```diff
@@ -83,9 +83,7 @@ We have automated e2e tests which keep CodeReady Containers in shape.
 
 [[intro-to-e2e-testing]]
 === Introduction
-End-to-end (e2e) tests use the link:http://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries.
-
-Clicumber allows running commands in a persistent shell instance (`bash`, `tcsh`, `zsh`, Command Prompt, or PowerShell), assert its outputs (standard output, standard error, or exit code), check configuration files, and so on. The general functionality of Clicumber is then extended by CodeReady Containers specific test code to cover the whole functionality of CodeReady Containers.
+End-to-end (e2e) tests borrow code from link:http://github.com/code-ready/clicumber[Clicumber] package to provide basic functionality for testing CLI binaries. This facilitates running commands in a persistent shell instance (`bash`, `tcsh`, `zsh`, Command Prompt, or PowerShell), assert its outputs (standard output, standard error, or exit code), check configuration files, and so on. The general functionality of Clicumber is then extended by CodeReady Containers specific test code to cover the whole functionality of CodeReady Containers.
```

Comment:
Shouldn't we update https://github.com/code-ready/clicumber codebase to use latest version and then consume it as we are doing now?

Reply:
For easier maintenance etc., the idea was to include it on our side instead. I mention #3210 in the description above to hint at this.

Reply:
@jsliacan which mean we are not using  https://github.com/code-ready/clicumber anymore?

Reply:
Correct, after this PR we won't be importing clicumber project.